JavaBytecodeCompiler/src/main/java/org/openzen/zenscript/javabytecode/compiler/JavaExpressionVisitor.java

```diff
@@ -29,6 +29,7 @@
 import java.util.*;
 import java.util.stream.Collectors;
 
+@SuppressWarnings("ALL")
```

Comment:
?

Reply:
I meant to remove it, added it there while I was debugging and forgot about it.